app/consent-form/_components/InfoText.tsx

```diff
@@ -0,0 +1,8 @@
+const InfoTextContent: string =
```

Comment:
i would pass this down as a prop if you want to separate it from the rest of the component, maybe with this text as a default value

Reply:
OOoo i like that idea